torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -119,11 +120,10 @@ std::shared_ptr<FutureMessage> RequestCallbackImpl::processRpc(
           stack.size());
 
       ownerRRef->setValue(std::move(stack.front()));
-      ctx.addForkOfOwner(
-          scriptRemoteCall.retRRefId(), scriptRemoteCall.retForkId());
-      return wrap(
-          RemoteRet(scriptRemoteCall.retRRefId(), scriptRemoteCall.retForkId())
-              .toMessage());
+      if (rrefId != forkId) {
```

Comment:
I feel we need some comment here to explain the difference of 2 cases

1) In what case, rrefId == forkId
2) why, in this case, don't need to add a fork